rmgpy/molecule/molecule.py

```diff
@@ -912,13 +953,15 @@ class Molecule(Graph):
     `inchi`                 ``str``     A string representation of the molecule in InChI
     `smiles`                ``str``     A string representation of the molecule in SMILES
     `fingerprint`           ``str``     A representation for fast comparison, set as molecular formula
+    `metal`                 ``str``     A represen
```

Comment:
Should add another line for facet as well?

Reply:
fixed